libp2p/NodeTable.h

```diff
@@ -276,11 +277,15 @@ class NodeTable : UDPSocketEvents
     Secret m_secret;												///< This nodes secret key.
 
     mutable Mutex x_nodes;											///< LOCK x_state first if both locks are required. Mutable for thread-safe copy in nodes() const.
-    std::unordered_map<NodeID, std::shared_ptr<NodeEntry>> m_allNodes;  ///< Known Node Endpoints
+    std::unordered_map<NodeID, std::shared_ptr<NodeEntry>>
+        m_allNodes;  ///< Node endpoints. Includes all nodes that we've been in contact with and
```

Comment:
Formatting like this is weird, move multi-line comments to the lines preceding the variable please.
(you should also remove `<` symbols then)

Reply:
Will do...this is automatic clang formatting btw, we should also look into updating our format config